config.toml

```diff
@@ -64,10 +64,15 @@ provider_type = "MbedCrypto"
 # (Required) Name of key info manager that will support this provider.
 key_info_manager = "on-disk-manager"
 
+# (Optional) Decide how large (in bytes) buffers inside responses from this provider can be.
+# Requests that ask for buffers larger than this threshold will be rejected. Defaults to 1MB.
+#buffer_size_limit = 1048576
```

Comment:
I think for this option it is fine if we only have one global config option declared in the `core_settings` and that work for all the service.
That way, you just have to add the option to the [`GlobalConfig`](https://docs.rs/parsec-service/0.3.0/parsec_service/utils/struct.GlobalConfig.html) and access it from there directly when you need it 💯

Reply:
I thought it might be a good idea to have a separate one for each provider in case the provider itself has specific limitations that we might want to accommodate for. Although thinking about it, I can't really come up with anything concrete off the top of my head. I'll change this.

Reply:
I guess as all providers are running with the same allocated memory, that's fine to assume that they would all have similar limitations! Also, did not want to increase the configuration complexity for such a small feature 😋

Reply:
SGTM!